examples/lock-app/cc13x2x7_26x2x7/args.gni

```diff
@@ -35,3 +35,5 @@ chip_enable_ota_requestor = true
 # BLE options
 chip_config_network_layer_ble = true
 chip_bypass_rendezvous = false
+chip_enable_rotating_device_id = true
```

Comment:
Most people will copy & paste examples, so our default was to disable rotating device id to make it a explicit user choice when compiling.

Reply:
Makes sense, I can test this PR with rotating device id = true, but set it to false in the final PR.

Reply:
Updated here: https://github.com/project-chip/connectedhomeip/pull/15367/commits/e5d41def8c65ec2f15371f497e5ce4e82ce4696c